meta-lmp-base/recipes-bsp/u-boot/u-boot-fio/lmp-common.cfg

```diff
@@ -0,0 +1,8 @@
+CONFIG_FIT=y
+CONFIG_USE_BOOTCOMMAND=y
+CONFIG_BOOTDELAY=-2
```

Comment:
I'm not sure about this option. When debugging, I often set it to a positive value for a particular board. Others may do the same. Having this option in a machine config file would be more convenient.

LGTM for the rest.